kong/plugins/aws-lambda/v4.lua

```diff
@@ -154,67 +149,64 @@ local function prepare_awsv4_request(tbl)
     headers[k] = v
   end
 
-  -- if this is an AWS request, sign it
-  if region then
```

Comment:
The change will break the fix for https://github.com/Kong/kong-plugin-aws-lambda/issues/60. Probably you should leave it there.